UITests/Responses/Receipts/PrepaidCardSecondaryResponse.json

```diff
@@ -4,8 +4,8 @@
     "limit": 10,
     "data": [
         {
-            "token": "trm-token",
-            "primaryCardToken": "trm-0000",
+            "token": "trm-token1",
+            "primaryCardToken": "trm-token1",
             "type": "PREPAID_CARD",
```

Comment:
since the primary card token is now trm-token2, can you update the primaryCardToken to use trm-token2?

Reply:
@malaw-hw should we not use `trm-token1` as primary and `trm-token2` as secondary?

Reply:
I have used it for the unique name in token for the ppc's, did not seen it as order or sequence.